cli/cmd/report_definitions_create.go

```diff
@@ -146,8 +146,20 @@ func promptCreateReportDefinitionFromNew() (reportDefinition api.ReportDefinitio
 
 	var sections []api.ReportDefinitionSection
 
+	cli.StartProgress("Fetching list of policy ids...")
+	resp, err := cli.LwApi.V2.Policy.List()
```

Comment:
Same here, can we filter by domain? There will be a lot of errors if the user selects the wrong policy